src/main/java/address/util/Version.java

```diff
@@ -53,25 +59,40 @@ public static Version fromString(String versionString) throws IllegalArgumentExc
             throw new IllegalArgumentException(String.format(EXCEPTION_STRING_NOT_VERSION, versionString));
         }
 
-        return new Version(Integer.parseInt(versionMatcher.group(1)),
-                Integer.parseInt(versionMatcher.group(2)),
-                Integer.parseInt(versionMatcher.group(3)));
+        if (versionMatcher.group(4) == null) {
+            return new Version(Integer.parseInt(versionMatcher.group(1)),
+                    Integer.parseInt(versionMatcher.group(2)),
+                    Integer.parseInt(versionMatcher.group(3)),
+                    false);
+        } else {
+            return new Version(Integer.parseInt(versionMatcher.group(1)),
+                    Integer.parseInt(versionMatcher.group(2)),
+                    Integer.parseInt(versionMatcher.group(3)),
+                    true);
+        }
     }
 
     @JsonValue
     public String toString() {
-        return String.format("V%d.%d.%d", major, minor, patch);
+        if (isEarlyAccess) {
+            return String.format("V%d.%d.%dea", major, minor, patch);
+        } else {
+            return String.format("V%d.%d.%d", major, minor, patch);
+        }
```

Comment:
Can reduce this block to one line

return String.format("V%d.%d.%d%s", major, minor, patch, isEarlyAccess ? "ea" : "");